ui-modules/utils/quick-launch/quick-launch.js

```diff
@@ -122,13 +122,17 @@ export function quickLaunchDirective() {
             if ($scope.app.config) {
                 $scope.configMap = $scope.app.config.reduce((result, config) => {
                     result[config.name] = config;
-                    const configValue = parseConfigValue(get(parsedPlan, `services[0][${BROOKLYN_CONFIG}][${config.name}]`));
+
+                    const planConfigValue = parseConfigValue((parsedPlan[BROOKLYN_CONFIG] || {})[config.name]);
+                    const catalogItemConfigValue = parseConfigValue((parsedPlan.services[0] && parsedPlan.services[0][BROOKLYN_CONFIG] || {})[config.name]);
+                    const configValue = planConfigValue || catalogItemConfigValue;
```

Comment:
this will return the `catalogItemConfigValue` if `planConfigValue` is set to `0` or `false`, won't it, and i think that isn't desired.

should this be:

```
let configValue = parseConfigValue((parsedPlan[BROOKLYN_CONFIG] || {})[config.name]);
if (typeof configValue == 'undefined') {
  configValue = parseConfigValue((parsedPlan.services[0] && parsedPlan.services[0][BROOKLYN_CONFIG] || {})
}
```

could also have

```
if (typeof configValue == 'undefined') {
  configValue = parseConfigValue(get(config, 'defaultValue', null))
}
```

then finally (replacing section below)

```
$scope.entityToDeploy[BROOKLYN_CONFIG][config.name] = configValue;
```

Reply:
Moreover, this returns `catalogItemConfigValue` if `planConfigValue` is set to `0` or `false` or `undefined`, or `null`, etc.